src/main/java/racingcarbasic/RacingCars.java

```diff
@@ -0,0 +1,22 @@
+package racingcarbasic;
+
+import java.util.List;
+
+public class RacingCars {
+    private final List<Car> carList;
+
+    public RacingCars(List<Car> carList) {
+        this.carList = carList;
+    }
+
+    public List<Car> moveCars(List<Car> carList) {
+        carList.stream()
+                .forEach(car -> {
+                    car.setMoveStrategy(new StraightMove());
+                    car.move();
+                });
```

Comment:
List는 stream 없이 forEach 사용이 가능할 것 같네요~

```suggestion
        carList.forEach(car -> {
                    car.setMoveStrategy(new StraightMove());
                    car.move();
                });
```